### PR TITLE
refactor: condense logging for a supply checker

### PR DESCRIPTION
**Pull request by kingpinXD**
# Description

Format logging for supply checker

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

File: zetaclient/zeta_supply_checker.go [hunk 1]
```diff
@@ -150,27 +149,47 @@ func (zs *ZetaSupplyChecker) CheckZetaTokenSupply() error {
...
 
+type ZetaSupplyCheckLogs struct {
+	Logger                   zerolog.Logger
+	AbortedTxAmounts         sdkmath.Int `json:"aborted_tx_amounts"`
+	ZetaInTransit            sdkmath.Int `json:"zeta_in_transit"`
+	ExternalChainTotalSupply sdkmath.Int `json:"external_chain_total_supply"`
+	ZetaTokenSupplyOnNode    sdkmath.Int `json:"zeta_token_supply_on_node"`
+	EthLockedAmount          sdkmath.Int `json:"eth_locked_amount"`
+	NodeAmounts              sdkmath.Int `json:"node_amounts"`
+	Lhs                      sdkmath.Int `json:"lhs"`
```
Comment (lumtis) [inline zetaclient/zeta_supply_checker.go:160]: What does it mean? We should use a more explicit name for the attribute

File: zetaclient/zeta_supply_checker.go [hunk 2]
```diff
@@ -150,27 +149,47 @@ func (zs *ZetaSupplyChecker) CheckZetaTokenSupply() error {
...
+	Rhs                      sdkmath.Int `json:"rhs"`
```
Comment (lumtis) [inline zetaclient/zeta_supply_checker.go:161]: Same commnent

File: zetaclient/zeta_supply_checker.go (hunk 1 above)
Comment (kingpinXD) [reply zetaclient/zeta_supply_checker.go:160]: The left-hand side and right-hand side are just for internal use.

Comment (lumtis) [reply zetaclient/zeta_supply_checker.go]: Even if it is for internal use, I think it is still better to make explicit name or provide comment explaining

**Review by lumtis — COMMENTED** [review]
Just responded in the comments, I think we should explain in the code what lhs and rhs are

**Pull request closed without merging**